### ISSUE TITLE
Volcano plot improvements

### ISSUE THREAD

**Issue by ajcammack** [ndcn/soda-ndcn#143]
The upgrades to the volcano plot (e.g. adding the options for unsaturation coloring) have been super useful and I think are really useful to show in presentations. The other volcano plot software I've found, at least for non-coders like myself, don't have these functionalities, so SODA is really great in this way.

Only issue is that the plots themselves are a little difficult to manipulate to make them presentation or publication ready. I wonder if a few improvements might be possible to implement to get them to that level. A few suggestions:

1) manual settings of axes and gridlines (rather just zoom in/out and autoscale)
2) turning on/off individual datapoint labels permanently (rather just hover over)
3) options to change color palates (sometimes the colors it chooses are difficult to see or have potential colorblind issues)
4) re-sizing of datapoints (if there are a lot of dots, it can be hard to see them all when they are too big)
5) toggle on/off lipids by number of double bonds (e.g. show only lipids with 4 or more double bonds)

**Comment by ajcammack** [comment]
Not sure if you've seen or ever used this one:
https://huygens.science.uva.nl/VolcaNoseR/?data=3;;X92R_v_2R__log2FC_;X92R_v_2R___logPvalue_;SampleID&vis=2;0.42;-0.2,0.2;1.3;significant;sig&can=0;TRUE;;PE%2016:0_16:1,PE%2016:0_20:2,PE%2016:0_20:3,PE%2016:0_22:4,PE%2016:0_22:6,PE%2016:1_18:0,PE%2018:0_20:2,PE%2018:0_20:3,PE%2018:0_22:4,PE%2018:1_18:2,PE%2018:1_20:1,PE%2018:1_20:2,PE%2018:1_20:3,PE%2018:1_22:6&layout=;;TRUE;-1.5,1.5;0,5;X;400;600&color=4;none&label=;;TRUE;log2(FC);-log(p-value);TRUE;24;14;9;3;TRUE&

It has some nice options for doing some of the things I mentioned above, like sizing the plot/axes/gridlines and changing colors

**Comment by ajcammack** [comment]
One thing that would be super cool and useful for me specifically would be the ability to highlight specific subsets of lipids only. For example, if I could make all the dots gray, but toggle on only the lipids that have 4 double bonds to be a different color and have labels. In my datasets, I see a clear shift in saturation with a loss in the highly unsaturated lipids, so trying to figure out the best way to visualize it in volcano plot format.

**Comment by dolivierj** [comment]
Yes these seem like interesting points to improve the volcano plots. Making the plots publication ready will be something to be worked on probably by the end of the year, keep you updated.
